### PR TITLE
Update js docs after the webpacker changes

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**
I will handle the Action Cable and Assets pipeline guide separately from this PR.

### CODE REVIEW

**Comment by sham-sheer** [comment]
HI

**Comment by sham-sheer** [comment]
HI

File: guides/source/asset_pipeline.md [hunk 1]
```diff
@@ -184,9 +184,8 @@ the file `scaffolds.css` (or `scaffolds.scss` if `sass-rails` is in the
 `Gemfile`.)
 
 For example, if you generate a `ProjectsController`, Rails will also add a new
-file at `app/assets/javascripts/projects.coffee` and another at
-`app/assets/stylesheets/projects.scss`. By default these files will be ready
-to use by your application immediately using the `require_tree` directive. See
+file at `app/assets/stylesheets/projects.scss`. By default these files will be
```
Comment (matass) [inline guides/source/asset_pipeline.md]: misspelled ? `these files` -> `this file` ?

Comment (prathamesh-sonpatki) [reply guides/source/asset_pipeline.md]: No, I missed updating it. Earlier there were two files, now only one. Will take care of it another round of review. Thanks 👍

**Pull request merged**